packages/mgt-element/src/components/baseProvider.ts

```diff
@@ -65,6 +66,17 @@ export abstract class MgtBaseProvider extends MgtBaseComponent {
   })
   public dependsOn: MgtBaseProvider;
 
+  /**
+   * The base URL that should be used in the graph client config.
+   *
+   * @memberof MgtMsal2Provider
+   */
+  @property({
+    attribute: 'base-url',
+    type: String
+  })
+  public baseUrl: GraphEndpoint;
```

Comment:
Should we validate the input here.?

```suggestion
  public get baseUrl(): GraphEndpoint {
    return this._baseUrl;
  }

  public set baseUrl(value: GraphEndpoint) {
    if (validateBaseURL(value)) {
      this._baseUrl = value;
      return;
    }
    throw new Error('Invalid base URL');
  }
```

Reply:
Yes. This makes it way better and leaner. Thank you!